components/scream/src/physics/rrtmgp/atmosphere_radiation.hpp

```diff
@@ -58,14 +58,50 @@ namespace scream {
             std::set<FieldIdentifier> m_required_fields;
             std::set<FieldIdentifier> m_computed_fields;
 
+            // Input and input/output fields
             std::map<std::string,const_field_type> m_rrtmgp_fields_in;
             std::map<std::string,field_type>       m_rrtmgp_fields_out;
 
+            template<typename T>
+            using view_type = field_type::view_type<T*>;
+
+            template<typename T>
+            using host_view_type = field_type::get_view_type<view_type<T>,Host>;
+
+            using host_view_in_type   = host_view_type<const_field_type::RT>;
+            using host_view_out_type  = host_view_type<      field_type::RT>;
+            std::map<std::string,host_view_in_type>   m_rrtmgp_host_views_in;
+            std::map<std::string,host_view_out_type>  m_rrtmgp_host_views_out;
+
+
+            std::map<std::string,const Real*>  m_raw_ptrs_in;
+            std::map<std::string,Real*>        m_raw_ptrs_out;
+
             std::shared_ptr<FieldInitializer> m_initializer;
 
             util::TimeStamp m_current_ts;
             ekat::Comm            m_rrtmgp_comm;
             ekat::ParameterList   m_rrtmgp_params;
+
+            // Keep track of number of columns and levels
+            int ncol;
```

Comment:
This is totally a matter of preference, but I think that most headers use `m_` to start any variable that is local to that process.  Like the `m_raw_ptrs_in` above.

Reply:
Oh, `m_` for "member variable". Cool, that's an easy fix, I'll implement.